src/main/java/com/attacktimer/AttackTimerMetronomeConfig.java

```diff
@@ -83,19 +82,59 @@ default Color NumberColor()
 		return Color.CYAN;
 	}
 
+	@ConfigItem(
+			position = 5,
+			keyName = "lastColor",
+			name = "Last Tick Color",
+			description = "Configures the color of tick number when it says 1",
+			section = TickNumberSettings
+	)
+	default Color LastColor()
+	{
+		return Color.WHITE;
```

Comment:
Let's default this to Color.CYAN. I do like the option to have a different color though.

Reply:
Okok, I was thinking the same possibility